tests/acceptance/StaticContentNagvigation.feature

```diff
@@ -0,0 +1,52 @@
+Feature: A user visit gigadb website
```

Comment:
There is typo in the name of the file, it should be ``StaticContentNavigation.feature`` instead of ``StaticContentNagvigation.feature``